index.html

```diff
@@ -1,348 +1,328 @@
                         </ul>
-                    </li>
+                    </div>
+                    <div class="green-btn">
+                        <a href="single-project.html">See Project</a>
+                    </div>
+                </div>
 
-                    <li class="grid-item skill-item">
-                        <div class="item-icon">
-                            <img src="assets/images/framework-icon.svg"
-                                alt="icon" />
+                <div class="grid-item">
+                    <div class="item-image">
+                        <!-- <img src="" alt="place image here" /> -->
+                    </div>
+                    <div class="item-description">
+                        <div class="item-title">
+                            <h2>Mult-Post Stories</h2>
+                            <h3>Gain + Glory</h3>
                         </div>
-                        <h3 class="item-title">Frameworks</h3>
-                        <ul class="item-langs">
-                            <li>Bootstrap</li>
-                            <li>Ruby on Rails</li>
-                            <li>RSpec</li>
-                            <li>Capybara</li>
-                            <li>Selenium</li>
+                        <ul class="technologies">
+                            <li>Javascript</li>
+                            <li>Ruby on rails</li>
+                            <!-- <li>HTML</li>
+                            <li>CSS</li> -->
                         </ul>
-                    </li>
+                    </div>
+                    <div class="green-btn">
+                        <a href="single-project.html">See Project</a>
+                    </div>
+                </div>
 
-                    <li class="grid-item skill-item">
-                        <div class="item-icon">
-                            <img src="assets/images/skills-icon.svg" alt="icon"
-                                />
+                <div class="grid-item">
+                    <div class="item-image">
+                        <!-- <img src="" alt="place image here" /> -->
+                    </div>
+                    <div class="item-description">
+                        <div class="item-title">
+                            <h2>Mult-Post Stories</h2>
+                            <h3>Gain + Glory</h3>
                         </div>
-                        <h3 class="item-title">Skills</h3>
-                        <ul class="item-langs">
-                            <li>Codekit</li>
-                            <li>Github</li>
-                            <li>Codepen</li>
-                            <li>Gitlab</li>
-                            <li>Terminal</li>
+                        <ul class="technologies">
+                            <li>Javascript</li>
+                            <li>Ruby on rails</li>
+                            <!-- <li>HTML</li>
+                            <li>CSS</li> -->
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. unused code as comments makes your code not clean. Kindly remove all unused code this will make your code mush noise, It is always a good practice to keep your code clean from such comments.